src/js/player.js

```diff
@@ -2487,17 +2485,54 @@ class Player extends Component {
  */
 Player.players = {};
 
+let navigator = window.navigator;
 /*
  * Player instance options, surfaced using options
  * options = Player.prototype.options_
  * Make changes in options, not here.
- * All options should use string keys so they avoid
- * renaming by closure compiler
  *
  * @type {Object}
  * @private
  */
-Player.prototype.options_ = globalOptions;
+Player.prototype.options_ = {
+  // Default order of fallback technology
+  techOrder: ['html5','flash'],
+  // techOrder: ['flash','html5'],
+
+  html5: {},
+  flash: {},
+
+  // defaultVolume: 0.85,
+  defaultVolume: 0.00, // The freakin seaguls are driving me crazy!
```

Comment:
Default volume was removed at some point but this got left behind. It can be removed.